src/Memory/Memory.js

```diff
@@ -1,29 +1,37 @@
 import React from "react";
-import { StyleSheet, Text, View } from "react-native";
-
-export default Memory = (props) => {
-  // const { id } = props.id;
-  // const { location } = props.location;
-  // const { description } = props.description;
-  // const { image } = props.image;
-  // const { song } = props.song;
-  // const { aromas } = props.aromas;
+import { StyleSheet, Text, View, Image } from "react-native";
+import Modal from "../UI/Modal";
 
+export default Memory = ({ memory }) => {
   return (
     <View style={styles.container}>
       <Text>MEMORY</Text>
-      {/* <Image source={require(`${image}`)}></Image>
-      <Text> {description} </Text>
-      <Text>{aromas}</Text> */}
+      {/* <Modal source={{ uri: `${memory.image}` }} style={styles.memoryImage}> */}
+        <View>
+          <Text> {memory.description} </Text>
+          <Text>{memory.aromas}</Text>
+        </View>
+      {/* </Modal> */}
```

Comment:
We should make sure to change the style of the modal to take up a larger portion of the page.  We want users to be able to see the material very clearly.